README.md

```diff
@@ -278,6 +282,42 @@ Example:
     staging: true
 ```
 
+### `verify-enable`
+
+**Default**: `true`
+
+The `verify-enable` setting controls whether or not the generated signatures and certificates are
+verified with the `sigstore verify` subcommand after all files have been signed.
+
+This is not strictly necessary but can act as a smoke test to ensure that all signing artifacts were
+generated properly and the signature was properly submitted to Rekor.
+
+Example:
+
+```yaml
+- uses: trailofbits/gh-action-sigstore-python@v0.0.2
+  with:
+    inputs: file.txt
+    verify-enable: false
+```
+
+### `verify-cert-email`
+
+**Default**: Empty
+
+The `verify-cert-email` setting controls whether to verify the Subject Alternative Name (SAN) of the
+signing certificate after signing has taken place. If it is set, `sigstore-python` will compare the
+certificate's SAN against the provided value.
+
+This setting only applies if `verify-enable` is set to `true`.
+
+```yaml
+- uses: trailofbits/gh-action-sigstore-python@v0.0.2
+  with:
+    inputs: file.txt
+    verify-cert-email: john.smith@example.com
+```
```

Comment:
I might be under-thinking this, but I'm pretty sure this won't work correctly on certificates that are minted from a GitHub Actions OIDC token -- the SAN for these certificates is (IIRC?) the repository URL, which isn't typed as an RFC 822 (email) SAN.